### ISSUE TITLE
App Not Working

### ISSUE THREAD

**Issue by bokrma** [AwinashGoswami/email-engine#1]
Please check and fix the app:
![Screenshot 2024-06-07 at 12 13 01](https://github.com/AwinashGoswami/email-engine/assets/13594276/e38ceeff-8fb9-472a-bc2f-c9aaf9e93f94)
![Screenshot 2024-06-07 at 12 13 12](https://github.com/AwinashGoswami/email-engine/assets/13594276/72bb1e0f-5ef4-40e7-85f0-55ced138eeab)
![Screenshot 2024-06-07 at 12 13 17](https://github.com/AwinashGoswami/email-engine/assets/13594276/e6d7b8ae-6a3e-44ec-a041-2fe1f4acb616)

**Comment by AwinashGoswami** [comment]
Yes, there was bit issue. I did very well-structured code but my node app could not connect to elastic search server. Do I need to update or am I done with evaluation from your side? Thank you for your response.

**Comment by AwinashGoswami** [comment]
Okay. I review it.
